test/objects.test.js

```diff
@@ -1,6 +1,8 @@
 import * as babel from '@babel/core';
 import path from 'path';
 
+const importStatement = `import clsx from'clsx';`;
```

Comment:
Please revert all changes in this file, the tests here shouldn't check this.

Reply:
As of right now the plugin will not convert the usage of `classnames` or `clsx` if there are no imports. That's the reason why I had to add it.